include/ed_navigation/kinect_navigation_plugin.h

```diff
@@ -0,0 +1,68 @@
+#ifndef ed_kinect_navigation_plugin_h_
+#define ed_kinect_navigation_plugin_h_
+
+#include <ed/plugin.h>
+#include <ed/world_model.h>
+
+#include "../../src/depth_sensor_integrator.h"
+
+/**
+ * @brief The KinectNavigationPlugin class
+ * ED plugin for de depth_sensor_integrator. Which uses the depth image to detect objects
+ */
+class KinectNavigationPlugin : public ed::Plugin
+{
+
+public:
+
+    /**
+     * @brief constructor
+     */
+    KinectNavigationPlugin() {}
+
+    /**
+     * @brief destructor
+     */
+    virtual ~KinectNavigationPlugin() {}
+
+    /**
+     * @brief configure
+     * @param config
+     * @
+     * lib: libed_kinect_navigation_plugin.so
+     * parameters:
+     *  depth_sensor_integration:
+     *      frame_id: /map
+     *      topic: /amigo/top_kinect/rgbd
+     *      num_samples: 640
```

Comment:
This is already an improvement, but frankly more an example than an explanation. I haven't got a clue what slope, distances and number of samples are meant, so how do I choose proper values? Consider the discussion that Albert started.

Reply:
I don't know this either, because I didn't write the code of the depth_sensor_integration, only the plugin.